internal/command/milestone_test.go

```diff
@@ -183,7 +183,7 @@ func TestCommands_GetMilestonesReached(t *testing.T) {
 			cache := gomap.NewCache[milestoneIndex, string, *MilestonesReached](
 				context.Background(),
 				[]milestoneIndex{milestoneIndexInstanceID},
-				cache.CacheConfig{Connector: "memory"},
+				cache.Config{Connector: "memory"},
```

Comment:
I have considered the possibility also, but I'm not sure how that will play with Viper. `enumer` is capable of creating `UnmarshalYAML`, so that;s fine. But AFAIK viper internally unmarhalls into a map and then uses the mapstructure package to set the values to the struct types.

I will give it a try and let you know.

Reply:
Yeah, so YAML didn't work. But using  it does `TextUnmarshaler`. The connector type is now an enum